### PR TITLE
CPDP-2606: added user type content for uk residential property exit page

### PR DESCRIPTION
**Pull request by AndyHWChung**

### CODE REVIEW

File: conf/messages [hunk 1]
```diff
@@ -968,6 +968,13 @@ didYouDisposeOfResidentialProperty.cyaChange=whether UK residential property was
 
 ukResidentCanOnlyReportResidential.title=You cannot use this service
 ukResidentCanOnlyReportResidential.p1=You told us you were a UK resident on the day you sold or disposed of the UK property. You can only use this service to report UK residential property.
+ukResidentCanOnlyReportResidential.agent.p1=You told us your client was a UK resident on the day you sold or disposed of the UK property. You can only use this service to report UK residential property.
+ukResidentCanOnlyReportResidential.trust.p1=You told us the trust was a UK resident on the day you sold or disposed of the UK property. You can only use this service to report UK residential property.
+ukResidentCanOnlyReportResidential.capacitor.p1=You told us the person was a UK resident on the day you sold or disposed of the UK property. You can only use this service to report UK residential property.
+ukResidentCanOnlyReportResidential.personalRep.p1=You told us the person was a UK resident on the day you sold or disposed of the UK property. You can only use this service to report UK residential property.
```
Comment (nubz) [inline conf/messages:974]: don't think this should say `on the day **you** sold or disposed of...` - or the capacitor one actually

Comment (nubz) [reply conf/messages:974]: not even sure about the trust one either - but never sure if trust can be called `you` (as the trustee)

Comment (AndyHWChung) [reply conf/messages:974]: whoops, I'll take a look again

Comment (AndyHWChung) [reply conf/messages]: fixed now

**Pull request merged**